src/lib/y2storage/yaml_writer.rb

```diff
@@ -383,12 +383,14 @@ def yaml_encryption(encryption)
     # @param filesystem [::Storage::BlkFilesystem]
     # @return [Hash{String => Object}]
     #
+    # rubocop:disable Metrics/CyclomaticComplexity
+    #
```

Comment:
This is the second pull request in a row in which we disable rubocop calls instead of trying to fix the issue. Not sure if using a tool and then ignoring its warning is the way to go.